guides/welfare/parental_leave.md

```diff
@@ -1,7 +1,17 @@
-# Parental Leave
+# Maternity, Paternity & Shared Parental Leave
 
-Our current parental leave package is two weeks at full pay followed by statutory allowance.
+When you welcome a new child (birthed or adopted), Made Tech encourages you to take leave, to welcome the child into the world.
```

Comment:
For context:

> 5% (3,540) of children looked after on 31st March 2016 were under 1 year old
13% (9,140) were aged between 1 and 4 years old
20% (14,090) were aged between 5 and 9 years old
39% (27,220) were aged between 10 and 15 years old
23% (16,460) were aged 16 and over

From - https://www.adoptionuk.org/press-media/adoption-facts-and-figures-england